### DISCUSSION TITLE
Does cobbler 3.3 support CentOS 7？

### DISCUSSION THREAD

**Discussion by Junxieshiguan** [cobbler/cobbler#2848] [Version specific questions]
Since our system is still CentOS 7 and cobbler 2.8 is no longer maintained, we hope to use the latest version of cobbler in CentOS 7. Can the latest version of cobbler be installed and used normally in CentOS 7? ThanKs

**Comment by SchoolGuy** [comment]
Well kind of. Packages are not possible since CentOS 7 does not package the full Python 3 stack. But in theory if it is an acceptable compromise for you, you can do a pure source install. However this has the clear disadvantage that you get everything from the internet. I believe depending on the rest of the server installation it would be easier to upgrade to CentOS 8 and then use the EPEL packages. Those in theory have zero-day patches which you will receive a few hours later.

**Reply by opoplawski** [reply]
You could try the cobbler 3.2.2 package from https://copr.fedorainfracloud.org/coprs/orion/cobbler/ and let me know how that goes.

I'm not sure I'm going to update that to 3.3 as I'm looking to keep the GUI around personally.

**Reply by Junxieshiguan** [reply]
I also hope to use CentOS 8, but the server systems we use now use CentOS 7. If we update to CentOS 8, it will cost a lot.
